src/util/address.rs

```diff
@@ -699,7 +672,9 @@ impl<W: fmt::Write> fmt::Write for UpperWriter<W> {
 }
 
 /// Extract the bech32 prefix.
-/// Returns the same slice when no prefix is found.
+///
+/// # Returns
```

Comment:
It feels weird to have what's usually a top-level title (single `#`) in the docs but all other Rust doc has it if I remember correctly. :man_shrugging:

Reply:
I don't fully understand this comment. I've also never written docs with the `# Returns` and `# Examples` sections so I looked them up and learned them for this PR. Is this comment suggesting that I'm doing something wrong, or that it's a strange convention in Rust to use a single `#`?

Reply:
I'm saying it's a strange convention in Rust to use single `#` :)

Basically in markdown `#` means top-level title and by definition there should be only one but Rustceans repeat it anyway.

Reply:
Oh yes, I agree. S'pose we can't be perfect.